www/index.php

```diff
@@ -13,7 +13,7 @@
     $peoplefinder = new UNL_Officefinder($options);
 } elseif (strpos($_SERVER['REQUEST_URI'], '/myinfo') !== false) {
     $peoplefinder = new UNL_PersonInfo($options);
-}  else {
+} else {
```

Comment:
Can you revert the changes in this file so the git history of this file stays clean